db/models/task.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { STRING } = require('sequelize');
+
+module.exports = db =>
+  db.define('tasks', {
+    description: STRING,
+    categories: STRING
+  });
+
+module.exports.associations = (Bounty, { Task, BountyTask }) => {
+  Task.belongsToMany(Bounty, { through: BountyTask });
+  Task.belongsTo(User, { as: 'creator' });
+  Task.belongsTo(User, { as: 'assignee' });
```

Comment:
I'm not actually sure how to set these associations now, will have to keep experimenting.